modelforge/tests/test_utils.py

```diff
@@ -53,8 +53,8 @@ def test_rbf(RBF):
 
     from .helper_functions import preparePairlist_for_single_batch, return_single_batch
```

Comment:
I think the original function needs to be renamed `prepare_pairlist_for_single_batch` and this import to be updated accordingly.

Reply:
thanks for fixing this!